packages/sdk-ui-ts/src/utils/helpers.ts

```diff
@@ -15,3 +16,15 @@ export const getTokenLogoWithVendorPathPrefix = (image: string) => {
 
   return path.join('/', 'vendor', '@injectivelabs', 'token-metadata', image)
 }
+
+export const getTokenTypeBaseOnAddress = (address: string): TokenType => {
+  if (address.startsWith('inj')) {
+    return TokenType.CW20
```

Comment:
Why is the native denom type CW20? We can maybe have another TokenType - bankDenom or NativeToken or something like that

Reply:
I wiped out this method since it's no longer useful. it is being removed from hub and explorer and replaced by just checking the `TokenType` property of a denom